### DISCUSSION TITLE
Linguist broke in my repository?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81197] [Repositories]
I need some help with Linguist, I made [this commit](https://github.com/EwanCroft/ewancroft.github.io/commit/6c45d00babfeb9253dea25aeedfd629be780cd9a) to my website repository's .gitattributes file, which has seemed to have broken Linguist?

All of my other repositories have a working Linguist graph, so I'm just confused as to why my website repo is different.

If anyone can help, thank you in advance.

**Comment by ananuness** [comment]
Hii, maybe what you're trying to do is:

```
# Exclude root-level files from Linguist detection
/** linguist-detectable=false

# Set files within /docs/ as detectable by Linguist
docs/** linguist-detectable=true

# Exclude /docs/images/ from Linguist detection
docs/images/** linguist-detectable=false
```

 I'm showing just the lines i made some changes, based on some other configs that i saw. You can test it and see if it works for you! ☺️

**Reply by ghost** [reply]
I've added your suggestion, how long does it take to apply?

**Reply by ananuness** [reply]
If you have Linguist installed locally, you can run Linguist against the repo. If you don't, you will need to wait for GitHub to run Linguist, but I think you would see by now if it worked for you!!

**Reply by ghost** [reply]
no luck. 😔

**Reply by ananuness** [reply]
oh, i'm sorry, now i can't figure out what the problem is... I thought it could be the syntax :(( is it showing any error message? Or just don't work?

**Reply by ghost** [reply]
I don't host Linguist, so i guess my website repo is glitched or something? Thank you for trying.

**Reply by ananuness** [reply]
Maybe! Or it's something really small. I hope you find a solution!
